### PR TITLE
C2LC-500:Mirror character when facing left.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-500](https://issues.fluidproject.org/browse/C2LC-500) for details.

### CODE REVIEW

**Comment by the-t-in-rtf** [comment]
I noticed that the character icon in the character position controls doesn't rotate or flip in the (simulated) iPad Pro on Safari.  It does work on all desktop browsers I've tested.  I'll have to compare the (very similar) code used for the scene character, which does rotate and flip on the iPad.

**Comment by the-t-in-rtf** [comment]
It also happens on Safari, which makes it much easier to troubleshoot.

**Comment by the-t-in-rtf** [comment]
Seems like [Safari does not support transforms on whole svgs, but does support them on groups](https://stackoverflow.com/questions/48248512/svg-transform-rotate180-does-not-work-in-safari-11).  Since the scene character transform is performed on a group, it works as expected.  I am attempting a light refactor of the character position icon to wrap it in a group.

File: src/CharacterPositionController.js [hunk 1]
```diff
@@ -100,11 +100,16 @@ class CharacterPositionController extends React.Component<CharacterPositionContr
     }
 
     getWorldCharacter() {
-        const transform = `rotate(${this.props.characterState.getDirectionDegrees() - 90} 0 0)`;
+        // We use a CSS approach because Safari doesn't support scale transforms whole svgs (and ignores the rotation
+        // if you include scaling in the transform).  See:
+        // https://stackoverflow.com/questions/48248512/svg-transform-rotate180-does-not-work-in-safari-11
+        const classNames = ["CharacterPositionController__character-column-character"];
```
[the remark this answers is not in the document]
Comment (the-t-in-rtf) [reply src/CharacterPositionController.js]: Good idea, and done.

**Comment by sbates-idrc** [comment]
Tony, the rotation and flipping work great. One thought that I did have is that we have 2 different implementations for the rotating and flipping. Right now they are consistent, but it could be easy for them to diverge and we could only catch that by manual testing. Would it be feasible to use a single implementation? Such as using the class approach for rotation in the scene as well as in the character positioning controls?

**Comment by the-t-in-rtf** [comment]
We explored various approaches, but couldn't find an approach that would work for the scene character.  The problem is that transforms (both CSS and SVG) are like stroke, fill, and can only be set once by which ever mechanism (attribute, CSS selector) takes precedence.  We agreed to use the class-based approach for the character position controls (to fix the issues in Safari) and keep the SVG-based approach for the scene character.  We also agreed to test all angles for both approaches.  That work has been done and this should be ready for further review.

**Pull request merged**